### PR TITLE
Update Build Configuration

### PR DESCRIPTION
**Pull request by AdrianAndersen**
- Use Node 18 in the Dockerfile and copy env variables
- Add docker compose dev file
- Update readme to cover setup and be in line with rezervo

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,10 +35,8 @@ ENV NODE_ENV production
 RUN addgroup --system --gid 1001 nodejs
 RUN adduser --system --uid 1001 nextjs
 
-# You only need to copy next.config.js if you are NOT using the default configuration
-# COPY --from=builder /app/next.config.js ./
 COPY --from=builder /app/public ./public
-COPY --from=builder /app/package.json ./package.json
+COPY --from=builder /app/.env.local ./
```
Comment (mathiazom) [inline Dockerfile:39]: This should not be included in the image, but rather mounted via Docker Compose

Comment (AdrianAndersen) [reply Dockerfile]: donzo

File: .env.local.example [hunk 2]
```diff
@@ -2,6 +2,8 @@ REVALIDATION_SECRET_TOKEN=
 
 # Hostname of backend storing user configurations
 CONFIG_HOST=https://{backendHost}
+# When running rezervo-web in Docker
+# CONFIG_HOST=http://host.docker.internal:{backendHost}
```
Comment (mathiazom) [inline .env.local.example:6]:
```suggestion
# CONFIG_HOST=http://host.docker.internal:{backendPort}
```

Comment (AdrianAndersen) [reply .env.local.example]: ah right. too much copy paste hehe. Fixed

File: README.md [hunk 3]
```diff
@@ -4,12 +4,47 @@
...
 
+1. Install dependencies using Yarn
+   ```shell
+   yarn install
+   ```
+2. Define your own `.env.local` from `.env.local.example`
+   ```shell
+   # If you want [on-demand revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#on-demand-revalidation), make sure to define `REVALIDATION_SECRET_TOKEN` in `.env.local`
+   cp .env.local.example .env.local
+   ```
```
Comment (mathiazom) [inline README.md:19]: Comment should be moved out of code block and included as a Markdown comment (`>`)

Comment (AdrianAndersen) [reply README.md]: Good point, fixed

**Review by mathiazom — APPROVED** [review]

**Pull request merged**